learning/features/2d/particle_systems_2d.rst

```diff
@@ -37,11 +49,113 @@ the relevant texture property:
 
 .. image:: img/particles2.png
 
-Physics variables
------------------
+Time Parameters
+===============
+
+Lifetime
+--------
+
+The time in seconds that every particle will stay alive. When lifetime
+ends, a new particle is created to replace it.
+
+Lifetime: 0.5
+
+.. image:: img/paranim14.gif
+
+Lifetime: 4.0
+
+.. image:: img/paranim15.gif
+
+One Shot
+--------
+
+When enabled, a Particles2D node will emit all of it's particles once 
+and then never again.
+
+Preprocess
+----------
+
+Particle systems begin with 0 particles emitted, then start emitting.
+This can be an inconvenience when just loading a scene and systems like
+a torch, mist, etc begin emitting the moment you enter. Preprocess is
+used to let the system process a given amount of seconds before it is
```

Comment:
amount -> number (you can't have an "amount of seconds", but you can have an "amount of time")